component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/internal/TOTPAuthenticatorServiceComponent.java

```diff
@@ -193,4 +194,34 @@ protected void unsetIdentityProviderManagementService(IdpManager idpManager) {
 
 		TOTPDataHolder.getInstance().setIdpManager(null);
 	}
+
+	/**
+	 * Set claim metadata management service implementation.
+	 *
+	 * @param claimManagementService ClaimManagementService
+	 */
+	@Reference(
+			name = "claimManagementService",
+			service = org.wso2.carbon.identity.claim.metadata.mgt.ClaimMetadataManagementService.class,
+			cardinality = ReferenceCardinality.MANDATORY,
+			policy = ReferencePolicy.DYNAMIC,
+			unbind = "unsetClaimMetadataManagementService")
+	protected void setClaimMetadataManagementService(ClaimMetadataManagementService claimManagementService) {
+
+		if (log.isDebugEnabled()) {
+			log.debug("claimManagementService set in IdentityMgtServiceComponent bundle");
```

Comment:
same here as well (tab -> 4 spaces)

Reply:
this change will be reverted after the migration is completed